radar/infrastructure_config/puppet.yml

```diff
@@ -0,0 +1,14 @@
+version: 1
+name: Puppet
+blip:
+  - version: "2.6"
+    ring: ASSESS
+description: |
+  [Puppet](https://puppet.com/) is an open-core software configuration management tool.
+rationale:
+  We need to look into configuration management tools.
+related:
+  - name: Ansible
+    file: ansible.html
+  - name: Chef
+    file: chef.html
```

Comment:
We will take a look at the other options as well. Just because SaltStack is in use, it does not automatically mean that is the best fit for us. Sorry for if I seem salty, but we need to keep our options open :)

Reply:
I think its more important to have relevant tech in the radar than to actually have a planned Assements phase for it.